app/clinic/views/dashboard.html

```diff
@@ -1,5 +1,14 @@
 <div ng-include src="'../../common/patient/header/views/header.html'"></div><!--patient header -->
 
+<div class="row bg-info-border content-border border-rouded-top" ng-init="getAlerts()" ng-show="flags.length !== 0">
+    <div class="col-sm-12" ng-repeat="flag in flags">
+        <div class="alert alert-warning alert-dismissible" role="alert">
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            <strong>Alerta:</strong> {{flag.message}}
```

Comment:
Hi @steliomo,
All the text on the pages should be translatable to en/pt. Eg this line `<strong>Alerta:</strong> {{flag.message}}` should be something like <strong>{{'COMMON_ALERT' | translate}}:</strong> {{flag.name | translate}}. You can configure messages using [files here](https://github.com/esaude/esaude-emr-poc/tree/master/poc_config/openmrs/i18n)